docs/pages/docs/guide/customize-the-cascade-layers.mdx

```diff
@@ -0,0 +1,67 @@
+import { Steps } from 'nextra/components'
+
+# Customize the Cascade Layers
+
+In some scenarios, you may need more control over the Nextra predefined CSS to
+avoid unintended overrides of styles within cascade layers. Below is an example
+of how `nextra-theme-docs` uses
+[postcss-import](https://github.com/postcss/postcss-import) to place predefined
+CSS into a specified cascade layer:
+
+<Steps>
+
+### Install `postcss-import`
+
+Install [`postcss-import`](https://github.com/postcss/postcss-import) and add it to `postcss.config.js`:
```

Comment:
```suggestion
Install `postcss-import` and add it to `postcss.config.js`:
```
Didn’t saw you already mentioned link in the first paragraph, we can remove from here